core/go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/fsnotify/fsnotify v1.7.0
 	github.com/golang/mock v1.6.0
 	github.com/open-feature/open-feature-operator/apis v0.2.38-0.20231117101310-726a7f714906
-	github.com/open-feature/schemas v0.2.8
+	github.com/open-feature/schemas v0.2.9-0.20240115143711-4a0be4f48816
```

Comment:
This made me realize when we renamed https://github.com/open-feature/schemas to https://github.com/open-feature/flagd-schemas, we never changed the go-mod name, and the package isn't being indexed properly at https://pkg.go.dev/github.com/open-feature/flagd-schemas, so we're force to rely on a specific sha instead of a proper release here.

That's totally fine for this PR, but I will fix this issue.

cc @Kavindu-Dodan @beeme1mr

Reply:
@toddbaert could you create and link a ticket to track that work, please?

Reply:
[done](https://github.com/open-feature/flagd-schemas/issues/125)